xample-malware/gotbotkr.md

```diff
@@ -13,12 +13,46 @@ Modified version of a publicly available backdoor name GoBot2. Modifications are
+|[Defense Evasion::Indicator Removal on Host (T1070)](https://attack.mitre.org/techniques/T1070/)|GoBotKR removes the Zone identifier from the ADS (Alternate Data Streams) of the file, to conceal the fact the file has been downloaded from the internet. [[1]](#1)|
+|[Defense Evasion::Masquerading (T1036)](https://attack.mitre.org/techniques/T1036/)| GoBotKR uses filenames and registry key names associated with legitimate software. [[1]](#1)|
+|[Discovery::Software Discovery: Security Software Discovery (T1518.001)](https://attack.mitre.org/techniques/T1518/001/)|GoBotKR checks for processes associated with security products and debugging tools, and terminates itself if any are detected. It can enumerate installed antivirus software using the wmic command. [[1]](#1)|
+|[Discovery::System Information Discovery (T1082)](https://attack.mitre.org/techniques/T1082/)|GoBotKR uses wmic, systeminfo and ver commands to collect information about the system and the installed software. [[1]](#1)|
+|[Discovery::System Network Configuration Discovery (T1016)](https://attack.mitre.org/techniques/T1016/)|GoBotKR uses netsh and ipconfig to collect information about the network configuration. It has used Naver and Daum portals to obtain the client IP address. [[1]](#1)|
+|[Discovery::System Owner/User Discovery (T1033)](https://attack.mitre.org/techniques/T1033/)|GoBotKR uses whoami to obtain information about the victimized user. It runs tests to determine the privilege level of the compromised user. [[1]](#1)|
+|[Discovery::System Time Discovery  (T1124)](https://attack.mitre.org/techniques/T1124/)| GoBotKR can obtain the date and time of the compromised system. [[1]](#1)|
+|[Lateral Movement::Remote File Copy  (T1105)](https://attack.mitre.org/techniques/T1105/)| GoBotKR attempts to copy itself into public folders of cloud storage services (Google Drive, Dropbox, OneDrive). [[1]](#1)|
+|[Lateral Movement::Replication Through Removable Media (T1091)](https://attack.mitre.org/techniques/T1091/)|GoBotKR can drop itself onto removable media and relies on Autorun to execute the malicious file when a user opens the removable media on another system. [[1]](#1)|
+|[Command and Control::Connection Proxy (T1090)](https://attack.mitre.org/techniques/T1090/)|GoBotKR can be used as a proxy server. [[1]](#1)|
+|[Command and Control::Data Encoding (T1132)](https://attack.mitre.org/techniques/T1132/)|The communication with the C&C server is base64 encoded. [[1]](#1)|
+|[Command and Control::Remote File Copy (T1105)](https://attack.mitre.org/techniques/T1105/)|GoBotKR can download additional files and update itself. [[1]](#1)|
+|[Command and Control::Standard Application Layer Protocol (T1071)](https://attack.mitre.org/techniques/T1071/)|GoBotKR uses HTTP or HTTPS for C&C. [[1]](#1)|
+|[Command and Control::Uncommonly Used Port (T1571)](https://attack.mitre.org/techniques/T1571/)|GoBotKR uses non-standard ports, such as 6446, 6556 and 7777, for C&C. [[1]](#1)|
+
 Behaviors
```

Comment:
As a counterpart to "ATT&CK Techniques", these sections for all examples should be labeled "MBC Behaviors" - to make it clear that the distinction is MBC vs. ATT&CK.